.github/file-filters.yml

```diff
@@ -52,7 +51,6 @@ backend_build_changes: &backend_build_changes
 
 backend: &backend
   - *backend_build_changes
-  - *backend_dependencies
```

Comment:
Is this removed because it's duplicated? `**/*.pysnap` also seems to be duplicated.

Reply:
yeah, i deduped it